src/main/java/seedu/bigpp/parser/Parser.java

```diff
@@ -42,6 +43,8 @@ private Command parseMainMenuCommand(String userInput) {
             return new EnterViewerCommand();
         case "tutorial":
             return new EnterTutorialCommand();
+        case "back":
```

Comment:
instead of one "back" case for each menu, since it is a common back command, we could possibly check for this before we do the check on line 14 UI.getUiState()

Reply:
Amazing idea, you are so smart